src/App.js

```diff
@@ -1,38 +1,26 @@
 import 'react-native-gesture-handler';
-import React, { useState, useEffect } from 'react';
-import { View, ActivityIndicator, StyleSheet } from 'react-native';
+import React, { useEffect } from 'react';
 import { Provider } from 'react-redux';
 import { NavigationContainer } from '@react-navigation/native';
 import { AppearanceProvider, useColorScheme } from 'react-native-appearance';
+import RNBootSplash from 'react-native-bootsplash';
+import { enableScreens } from 'react-native-screens';
 import { store, persist } from './reducers';
 import Navigation from 'components/navigation';
 import { ColorScheme, DarkTheme, LightTheme } from 'helpers/Themes';
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-  },
-});
+enableScreens();
 
 export default function App() {
-  const [ready, setReady] = useState(false);
   const scheme = useColorScheme();
 
   useEffect(() => {
     persist(() => {
-      setReady(true);
+      RNBootSplash.hide({ duration: 500 });
```

Comment:
Is the animation customizable? It doesn't look native on iOS, at least in the video. Also, now we are not waiting for the persist store to be loaded, just a random 500 milliseconds?

Reply:
We still wait for the persisted state to load before hiding the splash screen. The `hide` method is called from the persistor callback, the duration is only to make the splash fading out. We can customize de animation doing something like [this](https://github.com/zoontek/react-native-bootsplash/blob/master/example/App.js).

Reply:
Also, while I agree with you about not being 100% necessary for iOS. This provides only one entry point to customize splash for both platforms.

Reply:
Yeah, I don't think the fade animation is ok for iOS, it doesn't look native. Maybe we could check for platform and hide without animation for iOS? I agree with the other arguments about keeping it

Reply:
I checked and it looks better without the animation in both platforms, so I removed it